json-rpc/types/src/views.rs

```diff
@@ -1442,22 +1442,37 @@ pub struct TowerStateResourceView {
     pub count_proofs_in_epoch: u64,
     pub epochs_validating_and_mining: u64,
     pub contiguous_epochs_validating_and_mining: u64,
-    pub epochs_since_last_account_creation: u64
+    pub epochs_since_last_account_creation: u64,
+    // ADDED FIELDS FROM ORIGINAL MOVE STRUCT
+    // the actual count of proofs in epoch considering the lazy computation
+    pub actual_count_proofs_in_epoch: u64
 }
 
 impl TryFrom<AccountState> for TowerStateResourceView {
     type Error = Error;
 
     fn try_from(state: AccountState) -> Result<TowerStateResourceView, Error> {
+      let this_epoch = match state.get_configuration_resource()?{
+          Some(cr) => cr.epoch(),
+          None => bail!("cannot get epoch data from account state"),
```

Comment:
This is the source of your autopay failure. I believe the issue is that the Account in question does not have a configuration resource. You have to get that resource from the root account.

Reply:
Nice catch!

I think this patched now with my last PR.